src/pages/index.js

```diff
@@ -67,11 +180,26 @@ const cardSection = new Section(
   ".elements"
 );
 
-cardSection.renderItems();
+api
+  .getInitialCards()
+  .then((response) => {
+    response.forEach((res) => {
```

Comment:
Needs correction: please don't iterate the cards here, you should render the cards using Section class. For example, you can initialize the section here and render the initial cards.

```js
// first declare section to use it when adding new cards
let cardSection;

api
  .getInitialCards()
  .then((response) => {
    cardSection = new Section(..);
    cardSection.renderItems();
```